Apps/Sandcastle/gallery/ScriptInInfoBox.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
Maybe rename this to `InfoBox JavaScript` or at least add spaces in the name.

Reply:
Done